### ISSUE TITLE
Unknown prop warning in validation example.

### ISSUE THREAD

**Issue by milanvodicka** [rofrischmann/react-controlled-form#28]
Getting this warning in validation example:
```
Warning: Unknown prop `initialFields` on <form> tag. Remove this prop from the element. For details, see https://fb.me/react-unknown-prop
    in form (created by Form)
    in Form (created by Connect(Form))
    in Connect(Form) (at index.js:30)
    in Provider (at index.js:29)
```

**Comment by robinweser** [comment]
This has been fixed with 1.2.2.

Should update the dependencies I guess.

**Comment by rofrischmann** [comment]
This has been fixed with 1.2.2.

Should update the dependencies I guess.

**Comment by milanvodicka** [comment]
Aha, well than issue is within examples ... check this out: https://github.com/rofrischmann/react-controlled-form/blob/master/examples/validation/package.json#L6 ... sry for misleading title.
